compiler/luci-interpreter/pal/cmsisnn/PALConv2d.h

```diff
@@ -136,12 +136,13 @@ static inline void ConvPerChannel(const tflite::ConvParams &params, const int32_
 }
 
 static inline void SetupScratchpadTensor(luci_interpreter::Tensor *scratchpad,
-                                         const luci_interpreter::DataType &data_type,
+                                         const luci_interpreter::DataType &input_data_type,
                                          const tflite::ConvParams &params,
                                          const tflite::RuntimeShape &input_shape,
                                          const tflite::RuntimeShape &filter_shape,
                                          const tflite::RuntimeShape &output_shape)
 {
+  (void)input_data_type;
```

Comment:
optional: you can add `assert(input_data_type == loco::DataType::S8);`

Reply:
Done